myhw-week1/created-data.js

```diff
@@ -0,0 +1,181 @@
+const mysql = require('mysql');
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'userdb'
+});
+
+connection.connect();
+
+// creating database world
+let sql = "create database if not exists world"
+connection.query(sql, function (error, results) {
```

Comment:
Query method is an asynchronous call. You might use this example as a guide for taking this into account: https://github.com/HackYourFuture/databases/blob/master/Week2/async-create-insert.js =)